src/include/mpir_request.h

```diff
@@ -367,6 +367,20 @@ MPL_STATIC_INLINE_PREFIX int MPIR_Request_completion_processing_fastpath(MPI_Req
 int MPIR_Request_completion_processing(MPIR_Request *, MPI_Status *, int *);
 int MPIR_Request_get_error(MPIR_Request *);
 
+extern int MPID_Request_is_anysource(MPIR_Request *);
+extern int MPID_Comm_AS_enabled(MPIR_Comm *);
```

Comment:
I don't think we need the `extern` keyword here and in general we don't use it (see two lines up).